src/containers/Modal/Modal.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { Modal as ModalComponent } from 'decentraland-ui'
+
+import { ModalProps } from './Modal.types'
+
+export default class Modal extends React.PureComponent<ModalProps> {
+  handleOnClose = () => {
```

Comment:
Following the [naming conventions](https://github.com/decentraland/standards/blob/master/standards/react-redux.md#21-components) this should be only `handleClose` 🤓